extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -503,7 +524,6 @@ export class ToolsAndEulaPage extends ResourceTypePage {
 	}
 
 	private enableDoneButton(enable: boolean) {
-		this._isDoneButtonEnabled = enable;
```

Comment:
I assume this is removed because we are not ever on last page so this is irrelevant, is that right?